compio-fs/src/pipe.rs

```diff
@@ -14,21 +14,20 @@ use crate::File;
 /// Creates a pair of anonymous pipe.
 ///
 /// ```
-/// use compio_fs::pipe::pipe;
+/// use compio_fs::pipe::anon_pipe;
 ///
 /// # compio_runtime::block_on(async {
-/// let (rx, tx) = pipe().unwrap();
+/// let (rx, tx) = anon_pipe().unwrap();
 ///
 /// tx.write_all("Hello world!").await.unwrap();
 /// let (_, buf) = rx.read_exact(Vec::with_capacity(12)).await.unwrap();
 /// assert_eq!(&buf, b"Hello world!");
 /// # });
 /// ```
-pub fn pipe() -> io::Result<(Receiver, Sender)> {
-    let mut fds = [-1, -1];
-    syscall!(pipe(fds.as_mut_ptr()))?;
-    let receiver = unsafe { Receiver::from_raw_fd(fds[0]) };
-    let sender = unsafe { Sender::from_raw_fd(fds[1]) };
+pub fn anon_pipe() -> io::Result<(Receiver, Sender)> {
```

Comment:
Why not just `anonymous` and `named`, and users can `compio_fs::pipe::anonymous` and `compio_fs::pipe::named`. `anon_pipe` is not really a good name IMHO.

Reply:
OK, and mark here. I'll continue this PR after #71 merged.

Reply:
Sure

Reply:
`anon_pipe` is changed to `anonymous`, but `named` is not provided. Users need to spawn threads to create a sender and receiver into one program. See example `named_pipe` for details.